pkgs/applications/misc/qtpass/default.nix

```diff
@@ -18,9 +20,12 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir $out/bin
     mv $out/qtpass $out/bin
+    install -D {,$out/share/applications/}qtpass.desktop
+    install -D artwork/icon.svg $out/share/icons/hicolor/scalable/apps/qtpass-icon.svg
+    runHook postInstall
   '';
```

Comment:
I have kept it separated because the current install phase is just a temporary fix until the projects install make target does the right thing. On the other side `postInstall` is here to stay and the place where wrapping is usually done. However I don't mind merging them, this way it does just look cleaner and more future proof to me.